static/authui/css/main.css

```diff
@@ -184,35 +183,37 @@ a.btn {
 
 .content {
   background-color: white;
+  display: flex;
+  flex-direction: column;
 }
 
 .logo {
-  /* The logo is intended to be 21:9 */
-  /* When padding-top is given as percentage, it is the percentage of the width of the container */
-  /* So 9 / 21 == 42.86% */
-  padding-top: 42.86%;
+  /* The logo is intended to be fixed height */
+  height: 90px;
+  margin: 6px;
   background-image: url("../image/ic_app_logo.png");
   background-position: center;
-  background-size: 66px 66px;
+  background-size: 60px 60px;
   background-repeat: no-repeat;
   background-color: #eaf1fc;
 }
 
-.authgear-logo {
-  height: 80px;
-  background-position: center;
-  background-size: 100px auto;
-  background-repeat: no-repeat;
-  background-image: url("../image/ic_footer_authgear.png");
+.pane {
+  box-shadow: 0px 3px 8px rgba(0, 0, 0, 0.25);
+  border-radius: 6px;
+  padding: 6px;
+  margin: 6px;
+}
+
+.nav-bar {
+  margin: 6px;
+  display: flex;
+  flex-direction: row;
 }
 
 .back-btn {
-  width: 36px;
-  height: 36px;
-  background-color: #f3f2f1;
-  border-radius: 18px;
-  background-image: url("../image/ic_back_arrow.png");
-  background-position: center;
+  /* background-image: url("../image/ic_back_arrow.png"); */
```

Comment:
This can be removed.

Reply:
Actually we are missing the back button icon here. But I could not export the icon from the design file. Let me add a FIXME here.